src/mailer/index.ts

```diff
@@ -3,15 +3,26 @@ import {
   signinMessage,
   resetMessage,
 } from './templates';
-const ride = process.env.VG_CONTACT_EMAIL || 'viking.garage.app@gmail.com';
-const config = {
-  service: process.env.MAIL_SERVICE,
+
+if (!process.env.GMAIL_USER ||
+    !process.env.GMAIL_CLIENT_ID ||
+    !process.env.GMAIL_CLIENT_SECRET ||
+    !process.env.GMAIL_REFRESH_TOKEN) {
+  console.log('You should set Gmail OAuth2 envs');
+}
+
+const transporter = nodemailer.createTransport({
```

Comment:
this one doesn't return any error (with wrong configuration error will be on `transporter.sendMail()`)